internal/cmdrpkgfindupdates/command.go

```diff
@@ -0,0 +1,221 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmdrpkgfindupdates
```

Comment:
This is more of note to self that this is getting out of hand :).

In alpha version of kpt, package structure was dictated with the desire to share command implementations between kpt and kustomize and that turned out to be bad idea because cmd implementations are closer to the UX of the tools and differ considerably. 

I think now we have very clear idea what needs to be shared and what not, we can refactor to simplify the structure. 

I have created https://github.com/GoogleContainerTools/kpt/issues/3297 to tackle this.